.github/workflows/performance-benchmark.yml

```diff
@@ -11,6 +11,8 @@ jobs:
   CI:
     continue-on-error: true
     strategy:
+      # We must avoid pushing to the benchmark repository concurrently.
```

Comment:
Because it will yields conflicts with version control?

Reply:
Yes, we clone the repository simultaneously in two builds and try pushing from both of them making one of them outdated before the push can complete.